tests/test_core.py

```diff
@@ -2478,11 +2489,12 @@ def test_dlfcn_qsort(self):
                 output_nicerizer=lambda x, err: x.replace('\n', '*'),
                 post_build=self.dlfcn_post_build)
 
-    if Settings.ASM_JS and SPIDERMONKEY_ENGINE and os.path.exists(SPIDERMONKEY_ENGINE[0]):
+    if Settings.ASM_JS and SPIDERMONKEY_ENGINE and os.path.exists(SPIDERMONKEY_ENGINE[0]) and not self.is_wasm():
       out = run_js('liblib.so', engine=SPIDERMONKEY_ENGINE, full_output=True, stderr=STDOUT)
       if 'asm' in out:
         self.validate_asmjs(out)
 
+  @no_wasm_backend('todo: dynamic linking')
```

Comment:
Oh right. I think what happened here and in the others is I temporarily disabled them for all wasm, then got them working again, and though I should keep them off for the wasm backend, forgetting we handle that more centrally. Fixing.